src/app/models/fractalis-models/fractalis-chart-description.ts

```diff
@@ -0,0 +1,14 @@
+/**
+ * Copyright 2017 - 2018  The Hyve B.V.
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+import {FractalisChartVariable} from './fractalis-chart-variable';
+
+export class FractalisChartDescription {
+  catVars?: FractalisChartVariable;
+  numVars?: FractalisChartVariable;
+}
```

Comment:
is this the format of FractalisChartDescription? I thought there would be more to it, and it differs for chart types

Reply:
For now these are the 2 we need, others can we added, when we decide to introduce the custom chart parameters panel. There are indeed different variable types returned for the survival plot only, so investigating/fixing is a part of [TMT-625](https://jira.thehyve.nl/browse/TMT-625)